libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -1465,6 +1481,17 @@ open class SalesforceSDKManager protected constructor(
         screenLockManager?.onAppForegrounded()
         (biometricAuthenticationManager as? BiometricAuthenticationManager)?.onAppForegrounded()
 
+        // Review push-notifications registration for the current user, if enabled.
+        userAccountManager.currentUser?.let { userAccount ->
```

Comment:
This is MSDK's internal call for one-time-on-foreground push notifications registration.  It's wrapped with a check for the current user since I believe that's the key condition.